CMakeLists.txt

```diff
@@ -74,7 +75,8 @@ find_package(PCL REQUIRED)
 find_package(Boost REQUIRED COMPONENTS system thread date_time python)
 
 find_package(APR REQUIRED)
-find_package(UIMA REQUIRED)
+find_package(MongoClientLibrary REQUIRED)
```

Comment:
this line should not be there any more....it certainly is not on the master branch